Riot/Managers/Theme/Themes/DarkTheme.swift

```diff
@@ -138,3 +139,13 @@ class DarkTheme: NSObject, Theme {
         button.setTitleColor(self.tintColor, for: .normal)
     }
 }
+
+//  MARK: - Theme v2
+
+extension DarkTheme: ThemeV2 {
+    
+    var colors: Colors {
+        return DarkColors()
```

Comment:
We are instantiating `DarkColors` each time we accessed it. Even if it is temporary we should at least add a fixme or todo commet here to make sure we will move it later.

Reply:
Converted to lazy variables.